guides/pricing_and_taxation.md

```diff
@@ -0,0 +1,127 @@
+
+Solidus can handle several prices for the same country, and will always select the most recently updated price.
+
+## Prices in the backend
+
+In the Solidus admin backend, prices will be shown and entered with or without VAT, depending on the configuration value `Spree::Config.admin_vat_country_iso`. This has to be set to the two-letter ISO code of the country your admin thinks in. For a store that mostly sells to Germany, this would be set to `DE`. The effect of this is that now all prices in the backend will assume to have the German VAT rate included.
+
+## Prices in the frontend
+
+When a customer first browses your store, we do not know which jurisdiction she lives in. We will, therefore, have to make an assumption about her whereabouts. This assumption is the `cart_tax_country_iso` property on the `Spree::Store` model - meaning that you can assume your customers to be from different countries depending on which store she browses.
+
+This way, you can have, for example, two different stores for France and Germany (`my-shop.com/fr` and `my-shop.com/de`), where each shows different prices to accustom for the different VAT rates in either country.
+
+***
+You can customize Solidus' pricing behaviour by creating a custom `Spree::Config.variant_price_selector_class` along with a fitting `Spree::Config.pricing_options_class`.
+See the [specifications](https://github.com/solidusio/solidus/blob/master/core/spec/models/spree/variant/price_selector_spec.rb) for the standard [Price Selector](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/variant/price_selector.rb) as well as the [specifications](https://github.com/solidusio/solidus/blob/master/core/spec/models/spree/variant/pricing_options_spec.rb) for the standard [Pricing Options](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/variant/pricing_options.rb) for inspiration.
+***
+
+## Examples
+
+### Sales Tax
+
+Sales tax is the default tax type for any tax rate in Solidus.
+
+Let's say you need to charge 5% additional ("Sales") tax for all items that ship to New York and 6% on clothing items that ship to Pennsylvania. This will mean you need to construct two different zones: one zone containing just the state of New York and another zone consisting of the single state of Pennsylvania.
```

Comment:
It's a technical document, and describes what's possible. Something like US sales tax is possible, and that's what the documentation tells you. Zipcode-level taxation is not possible, but there might be stores that do not require it... Maybe.